### ISSUE TITLE
[A11Y] Missing instructions/example to enter on the issues field in filters on submission dashboard

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9362]
## Issue Description
// What is the nature of the issue? How does it impact users?

When filtering submissions on the submission dashboard based on issues, there isn't instructions of possible data to enter neither a placeholder text to provide further information to users about what they can type in the field. This field works as a live suggestion list, and it should be warned in somehow as well.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Log in as a journal editor / journal manager / editor
2. In the submissions dashboard, click on filters
3. Go to the "filter by issue" option 

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4 

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method

- Manual: Keyboard

## Action
// Describe action that can be taken to remediate the issue. 

Add instruction text along the field and/or a placeholder text to be shown in the field

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/0de5ff42-c9ed-429e-96bc-6cda87559844)

**Comment by Devika008** [comment]
In 3.4 I propose to add the following placeholder in HexCode: 505050 on the form field - Type issue number here

As a part of 3.5 redesign of submission, I have changed the filter section to reflect this-

![image](https://github.com/pkp/pkp-lib/assets/112014397/ff3071d2-3480-486e-9d89-35bc9d035b1a)
